ado/ipacheckenum.ado

```diff
@@ -13,12 +13,15 @@ program ipacheckenum
 	#d ;
 	syntax varname using/ ,
 	    dkrfvars(varlist) 
-	    missvars(varlist) 
-	    durvars(varlist) 
+	    missvars(varlist)  
 	    subdate(varname)
-		[duration(varname) exclude(varlist) foteam(varlist) days(integer 7)]
+		[durvars(varlist) duration(varname) exclude(varlist) foteam(varlist) days(integer 7)]
 		[replace modify];
 	#d cr
+	*loc dkrfvars gender ward

```

Comment:
I know these are the variables for the test case but I don't think it's necessary to keep the commented code here.